src/components/vm/vmCloneDialog.tsx

```diff
@@ -78,7 +101,7 @@ export const CloneDialog = ({ name, connectionName }) => {
                     onClone();
                 }}
                 isHorizontal>
-                    {!isObjectEmpty(error) && <ModalError dialogError={error.dialogError} dialogErrorDetail={virtCloneOutput} />}
+                    {error.dialogError && <ModalError dialogError={error.dialogError} dialogErrorDetail={virtCloneOutput} />}
```

Comment:
This added line is not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-2187-561408b0-20250717-145115-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)